### ISSUE TITLE
Question about the model and your code

### ISSUE THREAD

**Issue by brennn1** [lbenning/Load-Forecasting#1]
Hi Luke,

First off - very nice work!  

I was wondering - why wouldn't I get 100% accuracy when I trained the model using the 2014 data, and also test using the 2014 data?

**Comment by lbenning** [comment]
Which model?

**Comment by brennn1** [comment]
I was playing around with SVR.  

I modified the Datagen function to look like:
files = ["data/2014.xls", "data/2014.xls"]

I have another question.  I have similar load data that is collected every hour.  But some of my load data contains negative numbers because of electricity generation from solar panels.  Do you suggest just adding an offset to my values to get them all positive (because you're using log scaling).

**Comment by lbenning** [comment]
Yeah, the offset might be good to try. Some of the models don't get 100%
since they don't perfectly fit the data, e.g. due to regularization.

**Comment by brennn1** [comment]
Do you happen to have the Python code where you tested against the traditional economic models?
